go.mod

```diff
@@ -4,7 +4,6 @@ go 1.19
 
 require (
 	buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go v1.32.0-20231115204500-e097f827e652.1
-	buf.build/gen/go/bufbuild/registry/protocolbuffers/go v1.32.0-20231205222057-ac336d436f46.1
```

Comment:
Now that bufcasalpha removed dependencies on the old storage APIs, we no longer require this dependency on main.